source/scripting_v3/GTA/Entities/Entity.cs

```diff
@@ -756,6 +756,15 @@ public float MaxSpeed
 			set => Function.Call(Hash.SET_ENTITY_MAX_SPEED, Handle, value);
 		}
 
+		/// <summary>
+		/// Gets the speed vector of this <see cref="Entity"/>.
+		/// </summary>
+		/// <param name="relativeToEntity"><see langword="true" /> to get the vector relative to this entity; <see langword="false" /> for relative to the world.</param>
+		public Vector3 SpeedVector(bool relativeToEntity)
```

Comment:
Should be `GetSpeedVector` since we typically name simple getter methods like that.